lib/request/rack.rb

```diff
@@ -136,6 +136,7 @@ def if_modified_since
     accessor(:host,         Key.new('SERVER_NAME'))
     accessor(:query_string, Key.new('QUERY_STRING'))
     accessor(:content_type, Key.new('CONTENT_TYPE'), nil)
+    accessor(:cookies,      Key.new('HTTP_COOKIE'), EMPTY_STRING)
```

Comment:
Why default to an empty string? As a client of this library I want to be able to determine if the header was present or not?